From 942d0cd249300365271b37d9173e05677f141368 Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Thu, 17 Oct 2019 07:25:17 +0000
Subject: [PATCH] riscv: increase stack size to 16KiB

There are posibilit that 8KiB is not enough and causes illegal
instruction while bringing up harts online.

See: https://lore.kernel.org/linux-riscv/mvm5zkrhe8q.fsf@suse.de/T/#t
---
 arch/riscv/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/riscv/Kconfig b/arch/riscv/Kconfig
index 01975d7c..85e15ebf 100644
--- a/arch/riscv/Kconfig
+++ b/arch/riscv/Kconfig
@@ -224,7 +224,7 @@ config XIP
 
 config STACK_SIZE_SHIFT
 	int
-	default 13
+	default 14
 
 config SPL_LDSCRIPT
 	default "arch/riscv/cpu/u-boot-spl.lds"
-- 
2.23.0

